djot/inline.lua

```diff
@@ -259,10 +259,10 @@ Parser.matchers = {
           if is_image then
             self:add_match(opener[1] - 1, opener[1] - 1, "image_marker")
             self:add_match(opener[1], opener[2], "+imagetext")
-            self:add_match(opener[4], opener[5], "-imagetext")
+            self:add_match(opener[4], opener[4], "-imagetext")
           else
             self:add_match(opener[1], opener[2], "+linktext")
-            self:add_match(opener[4], opener[5], "-linktext")
+            self:add_match(opener[4], opener[4], "-linktext")
```

Comment:
This is a bugfix for https://github.com/jgm/djot/issues/75

I am relatively confident  that is is correct, we only want to flag `]` as `-linktext`